pyproject.toml

```diff
@@ -49,8 +49,8 @@ lint = [
 ]
 
 [build-system]
-requires = ["pdm-pep517"]
```

Comment:
Could you point me to the info about the rename? The docs say that it is still "pdm-pep517" for the current version: https://pdm.fming.dev/latest/pyproject/build/ . `pdm build` works fine too with `pdm` 2.4.7.